lib/ansible/modules/identity/ipa/ipa_user.py

```diff
@@ -29,52 +30,69 @@
     choices: [ always, on_create ]
     version_added: 2.8
   givenname:
-    description: First name
+    description: First name.
+    type: str
   krbpasswordexpiration:
     description:
-    - Date at which the user password will expire
-    - In the format YYYYMMddHHmmss
-    - e.g. 20180121182022 will expire on 21 January 2018 at 18:20:22
+    - Date at which the user password will expire.
+    - In the format YYYYMMddHHmmss.
+    - e.g. 20180121182022 will expire on 21 January 2018 at 18:20:22.
     version_added: 2.5
+    type: str
   loginshell:
-    description: Login shell
+    description: Login shell.
+    type: str
   mail:
     description:
     - List of mail addresses assigned to the user.
     - If an empty list is passed all assigned email addresses will be deleted.
     - If None is passed email addresses will not be checked or changed.
+    type: list
+    elements: str
   password:
     description:
-    - Password for a user. Will not be set for an existing user unless C(update_password) is set to C(always), which is the default.
+    - Password for a user.
+    - Will not be set for an existing user unless C(update_password) is set to C(always), which is the default.
```

Comment:
```suggestion
    - Will not be set for an existing user unless I(update_password) is set to C(always), which is the default.
```

(or ``I(update_password=always)``)